css/main.css

```diff
@@ -1,3 +1,120 @@
 /* Add your CSS here */
 
-/* Dont' forget to link this file to your HTML in the <head> */
+/* Don't forget to link this file to your HTML in the <head> */
+@import url('https://fonts.google.com/specimen/Quicksand');
+@import url('https://fonts.google.com/specimen/Poppins');
+
+html,body {
+     margin: 0; 
+    padding: 0; 
+    box-sizing: border-box; 
+    font-family: 'Quicksand';
+   /* background-color: aqua; */
+}
+
+h1,h2,h3 {
+    font-family: 'Poppins';
+}
+
+input {
+    background: blue;
+}
+
+/*define grid*/
+.h-main {
+     display: grid; 
```

Comment:
When you disable this ruleset in Devtools, what happens to the layout? Do you need these rules?

You can  disable style rules by unchecking the box next to them in [Devtools](https://developer.chrome.com/docs/devtools/css/) .

<img width="741" alt="Screenshot 2022-11-18 at 17 34 08" src="https://user-images.githubusercontent.com/42203406/202814368-302862bd-ecbc-45e7-8bd7-441debce9ac0.png">

Reply:
I didn't realise this function in Devtools, I need to be familiar with it and how to use it .
Thanks for letting me know it.

Reply:
It's soooo useful. I use it all the time! Devtools are amazing. So helpful!